build/package/send-slack-notification.sh

```diff
@@ -15,6 +15,11 @@
 
 set -Eeou pipefail
 
+if [[ "${unstable-}" == "-unstable" ]]; then
+  echo "skip notification"
```

Comment:
only notify on releases, skip snapshots

Reply:
is this script called only with the release evergreen tasks? my understanding is that we are not calling this scrip as part of the package tasks (the ones that generate the snapshot). Did we change this behaviour? thanks!

Reply:
Discussed offline: we are not calling this script also with snapshots